src/ints/mouse.cpp

```diff
@@ -18,1549 +18,932 @@
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Load acceleration profiles not implemented");
-		break;
-	case 0x2c: // MS MOUSE v7.0+ - get acceleration profiles
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Get acceleration profiles not implemented");
-		break;
-	case 0x2d: // MS MOUSE v7.0+ - select acceleration profile
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Select acceleration profile not implemented");
-		break;
-	case 0x2e: // MS MOUSE v8.10+ - set acceleration profile names
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Set acceleration profile names not implemented");
-		break;
-	case 0x2f: // MS MOUSE v7.02+ - mouse hardware reset
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("INT 33 AX=2F mouse hardware reset not implemented");
-		break;
-	case 0x30: // MS MOUSE v7.04+ - get/set BallPoint information
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Get/set BallPoint information not implemented");
-		break;
-	case 0x31: // MS MOUSE v7.05+ - get current minimum/maximum virtual
-	           // coordinates
-		reg_ax = (uint16_t)mouse.min_x;
-		reg_bx = (uint16_t)mouse.min_y;
-		reg_cx = (uint16_t)mouse.max_x;
-		reg_dx = (uint16_t)mouse.max_y;
-		break;
-	case 0x32: // MS MOUSE v7.05+ - get active advanced functions
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Get active advanced functions not implemented");
-		break;
-	case 0x33: // MS MOUSE v7.05+ - get switch settings and accelleration
-	           // profile data
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Get switch settings and acceleration profile data not implemented");
-		break;
-	case 0x34: // MS MOUSE v8.0+ - get initialization file
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Get initialization file not implemented");
-		break;
-	case 0x35: // MS MOUSE v8.10+ - LCD screen large pointer support
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("LCD screen large pointer support not implemented");
-		break;
-	case 0x4d: // MS MOUSE - return pointer to copyright string
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Return pointer to copyright string not implemented");
-		break;
-	case 0x6d: // MS MOUSE - get version string
-		LOG(LOG_MOUSE, LOG_ERROR)("Get version string not implemented");
-		break;
-	case 0x53C1: // Logitech CyberMan
-		LOG(LOG_MOUSE, LOG_NORMAL)
-		("Mouse function 53C1 for Logitech CyberMan called. Ignored by regular mouse driver.");
-		break;
-	default:
-		LOG(LOG_MOUSE, LOG_ERROR)
-		("Mouse function %04X not implemented", reg_ax);
-		break;
-	}
-	return CBRET_NONE;
+    // This routine provides a function for mouse ballistics
+    //(cursor acceleration), to be reused by various mouse interfaces.
+    // Since this is a DOS emulator, the acceleration model is
+    // based on a historic PS/2 mouse specification.
+
+    // Input: mouse speed
+    // Output: acceleration coefficient (1.0f for speed >= 6.0f)
+
+    // If we don't have raw mouse input, stay with flat profile;
+    // in such case the acceleration is already handled by the host OS,
+    // adding our own could lead to hard to predict (most likely
+    // undesirable) effects
```

Comment:
I've tried it, and it's very nice, indeed! I prefer it to the non-raw mode for sure! I found the default sensitivity a bit too much for 320x200 games, though, but with a setting of 75-80 it's just perfect for me.